cypress/integration/collections_spec.js

```diff
@@ -0,0 +1,82 @@
+describe('Dashboard Collections Page', () => {
+  const host = process.env.DASHBOARD_HOST || 'http://localhost:3000/';
+
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit(`${host}#/collections`);
+      cy.url().should('include', '/#/auth');
+      cy.get('div[class=modal__internal]').within(() => {
+        cy.get('a').should('have.attr', 'href').and('include', 'token?');
+        cy.get('a').should('have.text', 'Login with Earthdata Login');
+      });
+    });
+  });
+
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.login(host);
+    });
+
+    it('displays a link to view collections', () => {
+      cy.visit(host);
+
+      cy.get('nav li a').contains('Collections').should('exist').as('collections');
+      cy.get('@collections').should('have.attr', 'href', '#/collections');
+      cy.get('@collections').click();
+
+      cy.url().should('include', 'collections');
+      cy.get('.heading--xlarge').should('have.text', 'Collections');
+
+      cy.get('table tbody tr').its('length').should('be.gt', 1);
+    });
+
+    it('collections page displays a button to add a new collection', () => {
```

Comment:
So now I think the AC aren't that good. Below I asked if the tests fulfilled the AC. The same thing occurs here.
`filling in the new collection form and clicking submit displays the list of collections with the new collection in it`


But clicking submit doesn't take you to the list of collections, it takes you to the collection detail page (/collections/collection/MOD09GQ/006) and you would have to follow to `Back to Collections` to see the added collection.

But is this even possible with the FAKE-API we're running?

Is there a way we can mock a new collection and add it and if we could, is that useful in any way?

Reply:
This also doesn't appear to pass the command line tests in circle (headless).  I tried it from my command line and saw it fail but also see that it works running cypress interactively.

Reply:
It's weired.  The command line worked for me.  But I added one more check to see if it helps.

Reply:
@laurenfrederick @kkelly51 The fake api server uses static data, and doesn't do anything with the POST/PUT/DELETE request.  Is it enough that we only test the interface?  To satisfy the AC, we need to use real api server.  Or we could update the fake api server to act like a real server, and I'm not sure how much work is there and if it's worth to do that.

Reply:
I'm not sure if you want to do this, but using Cypress you can actually stub routes on our fake API or in general. See https://docs.cypress.io/guides/guides/network-requests.html#Requests.

I'm doing this in [#499](https://github.com/nasa/cumulus-dashboard/pull/499/files#diff-18ed6a1b0092f55d3cb4dd831481484aR68).

Reply:
@markdboyd would this be instead of using the fake api? 

@jennyhliu  @flamingbear Is the problem that it doesn't take you back to the collection page or that the collection page does not have the new collection in it (or both)?

Reply:
@laurenfrederick In theory, yes, we could use that entirely instead of the fake API. However, I was just using it to stub certain routes that aren't going to provided as static fixtures (e.g. DELETE). 

Ultimately, I think it may be better to go with @jennyhliu's approach of just adding DELETE support to the fake API as long as we're going to use that

Reply:
@jennyhliu After the new collection `MOD09GQ/006` is added, it displays the collection page for the new collection.  I can add step to `Back to Collection` page if it's necessary.

Reply:
What I am thinking is we need to test that it's going to the "right" page, whatever page that is. I don't think the test that the collection page shows the new collection, since that's an API call and we're handling those with API unit and integration tests.

It just seems like the AC is off from what is actually happening on the dashboard?